.github/workflows/docker.yml

```diff
@@ -14,7 +14,7 @@ on:
         description: ''
         type: string
       platforms:
-        default: linux/amd64,linux/arm64
+        default: linux/amd64,linux/arm64/v8
```

Comment:
how those are related to docs update ?

Reply:
It's not, this is some left over after my testing, will remove it ASAP